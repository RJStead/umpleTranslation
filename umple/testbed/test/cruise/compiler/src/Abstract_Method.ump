generate Java "../src-gen-umple";

class AbstractMethod
{
    abstract;
    abstract void foo() {
        int a = 1 + 1;
    }
}
