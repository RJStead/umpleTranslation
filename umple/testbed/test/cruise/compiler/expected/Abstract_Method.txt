Warning 1008 on line 6 of file 'Abstract_Method.ump':
Method syntax could not be processed. It has been considered as Extra Code
Abstract_Method.ump:6: error: abstract methods cannot have a body
  abstract void foo () 
                ^
1 error